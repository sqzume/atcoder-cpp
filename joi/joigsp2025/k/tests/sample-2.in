14
1 2 2 1 2 1 1 2 1 2 2 1 1 1
5
1 2
1 14
5 11
3 12
4 7
