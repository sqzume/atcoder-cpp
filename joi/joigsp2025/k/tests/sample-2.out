0
8
4
6
2
